spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  before do
+    @list = Menu.new
```

Comment:
Here we're using the actual `Menu` class which means the tests aren't isolated. Dependency injection could help us isolate the classes here: https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md